### ISSUE TITLE
Problems running example backtest

### ISSUE THREAD

**Issue by xenmind** [EliteQuant/EliteQuant_R#1]
Hi,

I've tried running the example backtest on AMZN. First I get an error with PerformanceManager$public_methods$UpdatePerformance regarding a Browser() function call within it. I noted that the github source doesn't have Browser(), so removing this allows the backtest to start, which then leads to this error:

engine$Run()
[1] "Running Backtest ..."
[1] "long 2011-01-13 short.sma 176.25699914 long.sma 145.27964985"
Error in NextMethod(.Generic) : replacement has length zero

It looks like an error relating to merging xts objects before performing math operations on the result?

Any help would be appreciated,

Thanks,
Gavin.

**Comment by xenmind** [comment]
Hi,

Gave it another go. When the Browser fn fired, I just kept pressing enter and it stepped forward in the backtest each period. I had to press enter for every period and no trade info was reported to screen, which is not ideal, but it worked, there was an equity curve excel file in the output directory.

Gavin.

**Comment by glidis** [comment]
Same problem, so the Browser() is intentionally placed there?

**Comment by glidis** [comment]
I think you are right. 
Sourcing those '*.R' files manually, the example really works.
Perhaps removing the 'browser()' in 
`namespace:PerformanceManager$public_methods$UpdatePerformance` could end all these problems.

**Comment by EliteQuant** [comment]
Yes I use Browser() to debug. @xenmind if you know a new way to debug, would you mind letting me know. I started using Python mostly because rstudio debugging pain. Thanks.

**Comment by xenmind** [comment]
I use Rstudio debugging myself so I can't recommend anything else sorry. The problem seemed to be leaving the Browser() call in the R package backtest code. Looks like Glidis found the offending code to be removed.
